deliverables/README.md

```diff
@@ -1,14 +1,22 @@
 ## Deliverables
 #### Required
 ##### Questions
-- What is a framework?
+- What is a framework?  
+In computer programming, a software framework is an abstraction in which software providing generic functionality can be   selectively changed by additional user-written code, thus providing application-specific software.
+(The framework should follow good Model-View-Controller (MVC) separation principles.)
+
```

Comment:
Frameworks don't always adhere to the MVC paradigm. For example http://www.peej.co.uk/tonic/